.taskcluster.yml

```diff
@@ -61,9 +61,9 @@ tasks:
           workerType: batch
           payload:
             maxRunTime: 3600
-            image: python:3.9
+            image: python:3.9-slim
             command:
-              - "/bin/bash"
+              - "/bin/sh"
```

Comment:
This seems causing issues. See: https://community-tc.services.mozilla.com/tasks/VKo0l4RKRPiPUo9EwvfFCQ/runs/0/logs/public/logs/live.log#L38

Reply:
got it thank you. sorry for the trouble

Reply:
no trouble; thank you for the work!